docs/tutorials/01-intro.md

```diff
@@ -234,9 +234,9 @@ OCI container images built with `buildah` are completely standard as expected. S
 
     # dnf -y remove docker
 
-## Using Dockerfiles with Buildah
+## Using Dockerfiles/Containerfiles with Buildah
```

Comment:
Nit: Put Containerfiles first.  When referring to both we should always put Containerfiles first.